plugins/src/lib.rs

```diff
@@ -215,6 +215,19 @@ impl Parse for DeriveSerde {
     }
 }
 
+/// Generates:
+/// - derive of Debug, serde Serialize & Deserialize
+/// - serde crate annotation
+#[proc_macro_attribute]
+pub fn data(_attr: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
let's just call it `derive_serde` to be explicit about what it's doing.

Reply:
Done :)